components/brave_vpn/browser/connection/win/brave_vpn_helper/service_main.cc

```diff
@@ -37,6 +37,13 @@ bool ServiceMain::InitWithCommandLine(const base::CommandLine* command_line) {
     LOG(ERROR) << "No positional parameters expected.";
     return false;
   }
+
+  // Crash itself if crash-me was used.
+  if (command_line->HasSwitch(kBraveVpnHelperCrashMe)) {
+    CHECK(!command_line->HasSwitch(kBraveVpnHelperCrashMe))
```

Comment:
Q) this check failure doesn't make crash dump and upload?

Reply:
Ah, I think you want to test to make crash and dump loading is done properly with this switch.

Reply:
yes